src/js/pie.js

```diff
@@ -160,12 +169,18 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     floe.chartAuthoring.pieChart.pie.create = function (that) {
         var container = that.container,
             p = that.options.pieOptions,
-            width = p.width,
-            height = p.height,
+            width = p.width === "auto" ? floe.chartAuthoring.pieChart.pie.calcAutoscaleWidth(that) : p.width,
```

Comment:
As discussed, since "auto" settings for width and height don't help with the SVG re-scaling but introduce restrictions of consolidating sizes of the parent container and SVG itself, this "auto" setting could be removed from the implementation.